tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -227,6 +227,32 @@ def fe_input_user_data_repack(
     else:
         assert input_user_shapes is None
     # TODO: implement freeze_placeholder (issue 58560)
+    # At the moment set_tensor_value is implemented only for onnx
+    if framework == 'onnx' and freeze_placeholder is not None:
```

Comment:
please avoid limitation to support only ONNX FE. TF FE supports it as well. I don't know about PDPD, but anyway it should be responsibility of new frontends (without legacy) to notify users in case of limitations. In MO we should have generic code.